packages/manager/modules/pci/src/projects/project/storages/containers/containers.controller.js

```diff
@@ -59,10 +59,8 @@ export default class PciStoragesContainersController {
       let solution;
       if (!container.s3StorageType) {
         solution = OBJECT_CONTAINER_OFFERS_TYPES.SWIFT;
-      } else if (container.isHighPerfStorage) {
-        solution = OBJECT_CONTAINER_OFFERS_TYPES.HIGH_PERFORMANCE;
       } else {
-        solution = OBJECT_CONTAINER_OFFERS_TYPES.STORAGE_STANDARD;
+    solution = !container.s3StorageType ? OBJECT_CONTAINER_OFFERS_TYPES.SWIFT : OBJECT_CONTAINER_OFFERS_TYPES.S3;
```

Comment:
Test against `container.s3StorageType` is already made in the if, here you can be sure this ternary test will always return `OBJECT_CONTAINER_OFFERS_TYPES.S3`